jenkins/data-prepper/Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ubuntu2004-x64-docker-buildx0.6.3-qemu5.0-awscli1.22-jdk14'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(name: 'VERSION', description: 'The version of Data Prepper', trim: true)
```

Comment:
Instead of having this as a parameter can it come from the build itself?

Reply:
Presently, this isn't possible. But, I think it could be something we add in the future.